apps/client/src/components/Policy/Media/index.tsx

```diff
@@ -0,0 +1,183 @@
+'use client';
+
+import { BulletItem } from 'client/components';
+import { useGetWindowWidth } from 'client/hooks';
+
+import * as S from './style';
+
+const 설치대수설치위치및촬영범위 = [
+  '설치대수: 45대',
+  '설치 위치 및 촬영범위: 본교 건물 내ㆍ외부 및 주차장',
+];
+
+const 개인영상정보의확인방법및장소에관한사항 = [
+  '확인방법: 영상정보 관리책임자에게 미리 연락하고 개인영상정보(열람, 존재확인) 청구서로 요청한 후 확인 가능합니다.',
+  '확인장소: 행정실',
+];
+const 영상정보의안전성확보조치 = [
```

Comment:
```suggestion
];

const 영상정보의안전성확보조치 = [
```

사이 공백 두시면 더 깔끔해질 듯 합니다!

Reply:
수정되었습니다 90ce98496cfdf7fe4ad7b13a4498caf7683bc040